src/OrchardCore.Modules/OrchardCore.AdminDashboard/DashboardManageAttribute.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+namespace OrchardCore.AdminDashboard
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class DashboardManageAttribute : Attribute, IAsyncResourceFilter
```

Comment:
I would suggest just using an `httpContext.Feature` rather than needing another filter here.

Reply:
Updated code to use Feature